src/main/resources/assets/idwtialsimmoedm/lang/ru_ru.json

```
@@ -0,0 +1,47 @@
+{
+  "text.idwtialsimmoedm.config.title": "Настройки idwtialsimmoedm",
+  "text.idwtialsimmoedm.config.field.descriptionPrefix": "Префикс перед описанием (1 линия)",
+  "text.idwtialsimmoedm.config.field.descriptionIndent": "Отступы слева от описания (2+ линия)",
+  "text.idwtialsimmoedm.config.field.displayOnlyWhenShiftIsHeld": "Отображать описание только при удерживании Shift",
+  "text.idwtialsimmoedm.config.field.displayOnBooksOnly": "Отображать описание только в чародейских книгах",
+
+  "enchantment.minecraft.protection.desc": "Снижает урон от большинства источников.",
+  "enchantment.minecraft.fire_protection.desc": "Снижает урон от огня и время горения игрока.",
+  "enchantment.minecraft.feather_falling.desc": "Снижает урон от падения и телепортации с помощью эндер-жемчуга.",
+  "enchantment.minecraft.blast_protection.desc": "Снижает урон от взрывов и эффект откидывания при взрыве.",
+  "enchantment.minecraft.projectile_protection.desc": "Снижает урон от таких снарядов, как стрелы и огненные шары.",
+  "enchantment.minecraft.respiration.desc": "Улучшает задержку дыхания и видимость под водой.",
+  "enchantment.minecraft.aqua_affinity.desc": "Увеличивает скорость копания под водой.",
+  "enchantment.minecraft.thorns.desc": "С некоторым шансом наносит урон атакующим.",
+  "enchantment.minecraft.sharpness.desc": "Увеличивает урон.",
+  "enchantment.minecraft.smite.desc": "Увеличивает урон против нежити, таких как зомби и скелеты.",
+  "enchantment.minecraft.bane_of_arthropods.desc": "Увеличивает урон против членистоногих, таких как пауки и чешуйницы.",
+  "enchantment.minecraft.knockback.desc": "Увеличивает силу отбрасывания.",
+  "enchantment.minecraft.fire_aspect.desc": "Поджигает цель.",
+  "enchantment.minecraft.looting.desc": "Увеличивает выпадение добычи с убитых существ.",
```

Общий стиль, смысловой паритет с исходным текстом
```suggestion
  "enchantment.minecraft.looting.desc": "Повышение вероятности выпадения редкой добычи и её количества.",
```